nengo/_vendor/scipy/sparse/README.md

```diff
@@ -0,0 +1,6 @@
+linalg_expm
+-----------
+This code for the matrix exponential is taken from `scipy.sparse`.
+
+One function that could not be ported over (it is in C) is `scipy.special.comb`.
```

Comment:
scipy has a Python implementation (https://github.com/scipy/scipy/blob/v1.3.0/scipy/special/_comb.pyx#L7) also see comment below.

Reply:
Done in #1535